src/solvers/refinement/string_builtin_function.cpp

```diff
@@ -187,6 +187,25 @@ exprt string_set_char_builtin_functiont::length_constraint() const
     equal_exprt(return_code, return_value));
 }
 
+optionalt<exprt> string_to_lower_case_builtin_functiont::eval(
```

Comment:
I don't understand what the role of the `get_value` function is. Could you please document? (probably in `string_builtin_functiont`.